### PR TITLE
[R4R]add query option to getTokens api, resolve issue #57

### PR DESCRIPTION
**Pull request by guagualvcha**

### CODE REVIEW

File: common/types/query.go [hunk 1]
```diff
@@ -328,3 +328,30 @@ func (param *MarketsQuery) Check() error {
...
+	return param
+}
+
+func (param *OrdersQuery) WithLimit(limit uint32) *OrdersQuery {
+	param.Limit = &limit
+	return param
+}
+
+func (param *OrdersQuery) Check() error {
+	if param.Limit != nil && *param.Limit <= 0 {
```
Comment (yutianwu) [inline common/types/query.go]: should we check offset

File: common/types/query.go [hunk 2]
```diff
@@ -328,3 +328,30 @@ func (param *MarketsQuery) Check() error {
 	}
 	return nil
 }
+
+// OrdersQuery definition
+type OrdersQuery struct {
+	Offset *uint32 `json:"offset,omitempty,string"` //Option
+	Limit  *uint32 `json:"limit,omitempty,string"`  //Option
```
Comment (HaoyangLiu) [inline common/types/query.go]: Why do you use pointer?

File: common/types/query.go (hunk 1 above)
Comment (guagualvcha) [reply common/types/query.go]: yes, added.

File: common/types/query.go (hunk 2 above)
Comment (guagualvcha) [reply common/types/query.go]: The field is optional, use can not assign them so that will use default by backend service.

**Pull request merged**